.github/compute-version/action.yaml

```diff
@@ -0,0 +1,16 @@
+name: Build and Publish Images
+description: 'Publishes operator and bundle images to an Image Registry'
```

Comment:
Can we change the name and description of this workflow to align with computing the version?

Reply:
Good catch :)